docs/specs/clients/notify/authentication.md

```diff
@@ -10,6 +10,8 @@ All of the authentication payloads are DID JWTs and share the following claims:
 - iss - did:key. Value defined specifically in each payload
 - sub - did:pkh of blockchain account that this request is associated with
   - Example: `did:pkh:eip155:1:0x1234...`
+- mjv - major version of the API level being used as a string, currently `"1"`
```

Comment:
No specific/strong reason I choose this word, I think schema version would work. Not sure yet if each JWT might have different version #s or if it would be for the entire protocol, depends on how we end up using it. But for now we need to differentiate between `0` and `1` for the `sbs` response

```
mjv - major version of the schema being used as a string, currently `"1"`
```